Modules/System/Email/src/Scenario/EmailScenarioImpl.Codeunit.al

```diff
@@ -209,6 +209,7 @@ codeunit 8892 "Email Scenario Impl."
     procedure GetAvailableScenariosForAccount(EmailAccount: Record "Email Account Scenario"; var EmailScenarios: Record "Email Account Scenario")
     var
         Scenario: Record "Email Scenario";
+        EmailScenario: Enum "Email Scenario";
```

Comment:
We're probably going to do this change without the variable...

Reply:
Great :) I just did not know how :)